python/Ganga/__init__.py

```diff
@@ -299,27 +299,13 @@ def getLCGRootPath():
 lcg_config = makeConfig('LCG', 'LCG/gLite/EGEE configuration parameters')
 #gproxy_config = getConfig('GridProxy_Properties')
 
-# set default values for the configuration parameters
-lcg_config.addOption(
-    'EDG_ENABLE', False, 'DEPRECATED enables/disables the support of the EDG middleware')
-
-lcg_config.addOption('EDG_SETUP', '/afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh',
-                 'DEPRECATED sets the LCG-UI environment setup script for the EDG middleware',
-                 filter=Ganga.Utility.Config.expandvars)
-
-lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
```

Comment:
On 08/08/2016 14:01, Robert Currie wrote:
>
> In python/Ganga/__init__.py 
> <https://github.com/ganga-devs/ganga/pull/690#discussion_r73870496>:
>
> > @@ -299,27 +299,13 @@ def getLCGRootPath():
> >  lcg_config = makeConfig('LCG', 'LCG/gLite/EGEE configuration parameters')
> >  #gproxy_config = getConfig('GridProxy_Properties')
> >
> > -# set default values for the configuration parameters
> > -lcg_config.addOption(
> > -    'EDG_ENABLE', False, 'DEPRECATED enables/disables the support of the EDG middleware')
> > -
> > -lcg_config.addOption('EDG_SETUP', '/afs/cern.ch/sw/ganga/install/config/grid_env_auto.sh',
> > -                 'DEPRECATED sets the LCG-UI environment setup script for the EDG middleware',
> > -                 filter=Ganga.Utility.Config.expandvars)
> > -
> > -lcg_config.addOption(
> > -    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
>
> @drmarkwslater <https://github.com/drmarkwslater> I'm not sure what 
> you mean here, are you saying should we add a 2nd message to the 
> launch of Ganga after integrating the credentials branch?
>
Yes - partly to highlight the improvements we've put in (being able to 
submit jobs from different VOs in the same Ganga session/repo is quite a 
big thing) but also to tell people why they are seeing errors.

> I still think failing to load until people have read a message and 
> taken action is more likely to get people's attention and not to bug 
> those who don't use these features. (new users and people on local 
> batch systems). These errors should only be presented to people who 
> are directly effected by the changes in this PR from what I can tell.
>
Failing to load seems quite a major thing to do when Ganga can continue 
without issue. And the users don't *have* to take any action as these 
errors can be ignored. Plus, at present the release notes are only shown 
once so if you miss that or aren't reading the notes properly, you can't 
start Ganga and have just got an 'Unkown Config Option' error to go on, 
to which people may not think to just delete from their .gangarc. And 
actually, if this option is set on a site-wide/install level, the user 
may not be able to remove the option anyway if it's set in a .ini file 
somewhere.

@egede, @alexanderrichards  any thoughts?


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ganga-devs/ganga/pull/690/files/07b7161a099cd4ab74bceb760d60d671484d9cf0#r73870496>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ANCzOG-XKLoMOgmrJq-_Opbrqr3SG1t7ks5qdyi0gaJpZM4JT4Iv>.
>